clients/v1/clients.proto

```diff
@@ -102,6 +119,49 @@ message Role {
   optional string resource_type = 4;
 }
 
+message GetUserMetadataRequest {
+  // The user's external ID.
+  string id = 1;
+  // Services is the list of service slugs to filter the metadata. "*" can be used
+  // to return metadata for all services.
+  //
+  // Requires the scope for the requested service(s); for example, '*' requires
+  // 'sams::user.metadata::read', and 'dotcom' requires 'sams::user.metadata.dotcom::read'.
+  repeated string services = 2;
```

Comment:
@unknwon I realize that we call this "service" internally but it does not (and probably should not) align with "scope services", i.e. `services.Service`. Should we change the naming here to be more generic, like "namespace"?

Reply:
Renamed in 69d985475b17b43465f24001c3af5899f662efd3